### ISSUE TITLE
[Bug] iOS Accessibility Support 

### ISSUE THREAD

**Issue by AndyNetDuma** [xamarin/Xamarin.Forms#11647]
### Description
I have found many issues with iOS accessibility. I have an app that works fine on Android however on iOS it quickly falls apart.

- Tab order seems to be half ignored going to unnumbered controls first. Ie 0, 1, some unnumbered control, 2 etc. I didn't verify if it's just using the page order and my elements were roughly in order (similar to #11626)
- Changing control visibility doesn't update the accessibly tree so it's constantly focusing invisible elements (partially #9792)
- Z-order seems non-existent (partially #9792)

It's possible there are more but this was all I could see clearly

### Steps to Reproduce
Make a project with some controls and an visible overlay. Change it to invisible through a binding after launch and test with iOS voice over enabled.

### Expected Behavior
Accessibility works

### Actual Behavior
Issues listed above

### Basic Information

- Version with issue: 4.7.0.1179
- Last known good version: Haven't found one
- IDE: Visual studio
- Platform Target Frameworks: 
  - iOS:  Default, not sure how to tell what version that it
- Affected Devices: Every tested iOS device

### Reproduction Link
Unfortunately the code I'm working on is closed source. It should be fairly easy to recreate using info from all the other accessibility bugs.

### Workaround
I haven't found any viable workarounds and sadly need this support for accessibility certification

**Comment by PureWeen** [comment]
@AndyNetDuma can you test 4.8?

Can you attach a repro or based ?

or based on your comment here
> Unfortunately the code I'm working on is closed source. It should be fairly easy to recreate using info from all the other accessibility bugs.

Can I just close this issue because it sounds like the issues you are describing here are covered by "other accessibility bugs"

**Comment by AndyNetDuma** [comment]
> @AndyNetDuma can you test 4.8?
> 
> Can you attach a repro or based ?
> 
> or based on your comment here
> 
> > Unfortunately the code I'm working on is closed source. It should be fairly easy to recreate using info from all the other accessibility bugs.
> 
> Can I just close this issue because it sounds like the issues you are describing here are covered by "other accessibility bugs"

I would have to make a repro for it, I don't really feel it's needed as the bugs are so obvious.
If really needed I can try find some time but I am very busy so I can't promise anything.

I opened the issue because I felt the other issues, while similar, didn't actually cover my concerns. I also chose not to open it as separate issues as they are all closely linked and therefore would likely be solved together.

I should also note this issues is vital for anyone trying to achieve AA or AAA accessibility certification on iOS.

**Comment by jsuarezruiz** [comment]
@AndyNetDuma Did you reproduce #11626 on a device?.(It has been closed, and I am interested to know if we should reopen it or not).

**Comment by AndyNetDuma** [comment]
> @AndyNetDuma Did you reproduce #11626 on a device?.(It has been closed, and I am interested to know if we should reopen it or not).

I can't say either way. I don't think we use any listviews, only collection views and more often stacklayout bindable templates. So I didn't specifically recreate the issues just issues relating to tab index. I did recreate the issues above on a real device, I don't really use the iOS simulator to test.

Reading #11625 in more detail I suspect it's an issue in its own right because it's dealing with multiple levels of xaml scope while the issue I detail is just within 1 level.

So I would open it and try solve the iOS accessibility issues in parallel, it's hard to say how much overlap there is and it's possible the root cause is the same.

**Comment by PureWeen** [comment]
Closing for now as the issues referenced in the description seem to cover it

**Comment by AndyNetDuma** [comment]
I explained above that issues detailed here aren't entirely covered by the issues referenced in the description. If you really feel this issue should be closed the other issues need details adding to them to cover the concerns I have here.
